ethereum/core/src/main/java/org/hyperledger/besu/ethereum/core/encoding/receipt/TransactionReceiptEncodingConfiguration.java

```diff
@@ -15,7 +15,20 @@
 package org.hyperledger.besu.ethereum.core.encoding.receipt;
 
 public class TransactionReceiptEncodingConfiguration {
-  public static final TransactionReceiptEncodingConfiguration NETWORK = new Builder().build();
+
```

Comment:
Not sure I really like this class. There are some concepts mixed, like encoding format, and whether we encode the bloom filter. When we are doing eht69 we are not encoding bloom! 
Is this just used for encoding?